core-blocks/test/fixtures/core__pullquote__multi-paragraph.json

```diff
@@ -15,7 +15,7 @@
                                 "Paragraph ",
                                 {
                                     "type": "strong",
-                                    "key": "_domReact71",
+                                    "key": "_domReact73",
```

Comment:
I'm not sure why, but my changes made this test to fail even though they don't seem to be related. I changed the key for now so the tests pass again and will investigate it further in the following days.

Reply:
I created an independent PR to deal with this: #7448.